cli/command/trust/cli_test.go

```diff
@@ -0,0 +1,108 @@
+package trust
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/config/configfile"
+	"github.com/docker/cli/cli/trust"
+	"github.com/docker/docker/client"
+	notaryclient "github.com/docker/notary/client"
+)
+
+type notaryClientFuncType func(imgRefAndAuth trust.ImageRefAndAuth, actions []string) (notaryclient.Repository, error)
+
+// FakeCli emulates the default DockerCli
+type FakeCli struct {
```

Comment:
Isn't this a dupliate of `internal/test/cli.go:FakeCli`? I think all we need is `NotaryClient()` and a `SetNotaryClient(...)` added to the existing FakeCli

Reply:
sounds good, done!